Game4.py

```diff
@@ -0,0 +1,165 @@
+import os, sys
+import pygame
+from random import randint
+
+from misc import *
+
+
+class Game4Main:
+    """The Main Game4 Class - This class handles the main
+    initialization and creating of the Game."""
+
+    def __init__(self, width=480,height=360):
+        # initialize pygame
+        pygame.init()
+        # set window size
+        self.width = width
+        self.height = height
+        # create screen
+        self.screen = pygame.display.set_mode((self.width, self.height))
+        # create clock
+        self.clock = pygame.time.Clock()
+        self.score = 0
+
+
+    def mainLoop(self):
+        """Main screen for game."""
+        # initialize player
+        count = 0
+        play_len = 25
```

Comment:
It might be easier to follow the code if all the constants (or config variables) are all in one file. You can make a config.py class and put all the constants there, and load all the config variables at once when initializing the game. Also, it's a convention to use capital letters for constants e.g) `PLAY_LEN = 25`

Reply:
This convention (and others) come from <https://www.python.org/dev/peps/pep-0008/#constants>